gnd/src/main/java/com/google/android/gnd/model/feature/FeatureMutation.java

```diff
@@ -37,6 +37,8 @@
    */
   public abstract Optional<Point> getNewLocation();
 
+  public abstract Optional<ImmutableList<Point>> getNewPolygonVertices();
```

Comment:
`Optional<ImmutableList<Point>>` can be replaced with `ImmutableList<Point>`, to avoid having two "empty" states (list vs optional).

Reply:
updated.